client/components/MusicControl.tsx

```diff
@@ -6,8 +6,16 @@ import Slider from '@material-ui/core/Slider';
 import CurrentPlaying from '~components/CurrentPlaying/';
 import CurrentPlayingArt from '~components/CurrentPlayingArt/';
 import Rating from '~components/Rating/';
+import { flagSong } from '~logic/Song';
+import { AuthKey } from '~logic/Auth';
+import { toast } from 'react-toastify';
+import AmpacheError from '~logic/AmpacheError';
 
-const MusicControl: React.FC = () => {
+interface MusicControlProps {
+    authKey?: AuthKey;
```

Comment:
I just realized, this shouldn't be optional.

Reply:
changed :)